<ui version="4.0" >
 <class>ThresholdSettingFormClass</class>
 <widget class="QDialog" name="ThresholdSettingFormClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ThresholdSettingForm</string>
  </property>
  <widget class="QLabel" name="lbSlider" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>94</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbDecision" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>170</y>
     <width>128</width>
     <height>48</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbCancel" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>170</y>
     <width>129</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbDetailSetting" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>110</y>
     <width>129</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
